gensim/sklearn_integration/sklearn_wrapper_gensim_ldamodel.py

```diff
@@ -46,15 +46,6 @@ def __init__(
         self.gamma_threshold = gamma_threshold
         self.minimum_probability = minimum_probability
         self.random_state = random_state
```

Comment:
Please remove corpus parameter from constructor (you pass a corpus only for `fit*` methods) in both models.